beacon_node/eth2_libp2p/Cargo.toml

```diff
@@ -40,12 +40,15 @@ rand = "0.7.3"
 directory = { path = "../../common/directory" }
 regex = "1.3.9"
 strum = { version = "0.20", features = ["derive"] }
+superstruct = "0.2.0"
 
 [dependencies.libp2p]
 #version = "0.39.1"
 #default-features = false
+# TODO: Update once https://github.com/libp2p/rust-libp2p/pull/2103 and 
+# https://github.com/libp2p/rust-libp2p/pull/2137 are merged upstream.
```

Comment:
FYI, these have both been merged.

Reply:
Yeah, I figured we handle the update as part of #2493

https://github.com/sigp/lighthouse/pull/2300#discussion_r682204487